elements/bernardi-guzman-neilan-raugel.def

```diff
@@ -0,0 +1,50 @@
+name: Bernardi-Guzman-Neilan-Raugel
+html-name: Bernardi&ndash;Guzm&aacute;n&ndash;Neilan&ndash;Raugel
+min-degree: 1
+max-degree: 1
+short-names:
+  - BGNR
+ndofs:
+  triangle:
+    formula: 11
+  tetrahedron:
+    formula:
+    - k=1: 19
+    - k=2: 49
+categories:
+  - vector
+  - Hdiv
+  - macro
+reference-elements:
+  - triangle
+  - tetrahedron
+dofs:
+  edges: (if \(k>1\)) point evaluations in tangential directions at midpoints
+  facets:
+    - point evaluations in normal directions at vertices
```

Comment:
The element is always H^1 conforming, point evaluations at vertices most be associated to the vertices.